src/components/layout/LeftMenu.vue

```diff
@@ -18,21 +18,8 @@ export default {
   data() {
     return {
       menuItems: [
-        { label: this.$t("menu.contacts"), route: "/profiles/contacts" },
-        {
-          label: this.$t("menu.trailsTreasuries"),
-          route: "/trails/treasuries",
-        },
-        [
-          { label: this.$t("menu.amendBallots"), filter: "amend-ballots" },
-          { label: this.$t("menu.TFElection"), filter: "t-f-election" },
-          { label: this.$t("menu.polls"), filter: "polls" },
-          {
-            label: this.$t("menu.workerProposals"),
-            filter: "worker-proposals",
-          },
-        ],
-        { label: this.$t("menu.tokens"), route: "/tokens" },
+        { label: this.$t("menu.daos"), route: "/trails/ballots", filter: "polls" },
```

Comment:
DAOs label is supposed to replace Groups

Reply:
Also resolved in commit [d867ddc](https://github.com/telosnetwork/app-telos-native/pull/88/commits/d867ddcfefd9a5e13c6f1d22f7e87b3b1156065f).